lib/puppet/provider/rabbitmq_user/rabbitmqctl.rb

```diff
@@ -73,11 +74,8 @@ def tags=(tags)
   end
 
   def admin
-    if usertags = get_user_tags
-      (:true if usertags.include?('administrator')) || :false
-    else
-      raise Puppet::Error, "Could not match line '#{resource[:name]} (true|false)' from list_users (perhaps you are running on an older version of rabbitmq that does not support admin users?)"
-    end
+    raise Puppet::Error, "Could not match line '#{resource[:name]} (true|false)' from list_users (perhaps you are running on an older version of ra  bbitmq that does not support admin users?)" unless (usertags = get_user_tags)
```

Comment:
I'd do the assignment on a separate line. It's very long already. You also introduced spaces in "rabbitmq"